assignments/objects.js

```diff
@@ -19,27 +19,80 @@ const example = {
 
 // Write your intern objects here:
 
+const Mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const Kennan = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
+
+const Keven = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M",
+}
+
+const Gannie = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+
+const Antonietta = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+//console.log(Mitzi.name);
 
 // Kennan's ID
+//console.log(Kennan.id);
 
 // Keven's email
+//console.log(Keven.email);
 
 // Gannie's name
+//console.log(Gannie.name);
 
 // Antonietta's Gender
+//console.log(Antonietta.gender);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+Kennan.speak = function(){
+  console.log("Hello, my name is Kennan!")
```

Comment:
```suggestion
  return "Hello, my name is Kennan!"
```
You have to return the statement because on line 83 the console.log needs a value returned to console.log it